web/client/selectors/catalog.js

```diff
@@ -0,0 +1,17 @@
+const {get} = require('lodash');
+
+module.exports = {
+    resultSelector: (state) => get(state, `catalog.result`),
```

Comment:
why backticks?

Reply:
They are only needed here:
selectedServiceTypeSelector: (state) => get(state, `catalog.services.${get(state, "catalog.selectedService")}.type`, 'csw')

i'll replace them with use " in the other cases